ddl/ddl_worker.go

```diff
@@ -93,6 +109,38 @@ func (d *ddl) doDDLJob(ctx context.Context, job *model.Job) error {
 	}
 }
 
+func (d *ddl) writePreDDLBinlog(ctx context.Context, jobID int64, startTS uint64) error {
```

Comment:
Can this function be an async implementation? so write bin log and two-phase commit run parallel, latency would be lower.